CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+AWS Mobile SDK for iOS has removed these deprecated clients:
+
+- `AWSCognito` (Amazon Cognito Sync) - Use [`Amplify DataStore`](https://docs.amplify.aws/lib/datastore/getting-started/q/platform/ios) for data synchronization.
+- `AWSMobileAnalytics` - Use [`Amplify Analytics`](https://docs.amplify.aws/lib/analytics/getting-started/q/platform/ios) for analytics.
+- `AWSS3TransferManager` - Use [`Amplify Storage`](https://docs.amplify.aws/lib/storage/getting-started/q/platform/ios) for uploads and downloads with Amazon S3.
```

Comment:
We haven't really removed the s3 transfer manager, should we really list it here?

Reply:
We definitely are removing the [Transfer Manager](https://github.com/aws-amplify/aws-sdk-ios/pull/3281/files#diff-b9bf5cc90c919551b46841112de1173c3b9fb4579cdf9ad7b3f38815cbf59560). Are you perhaps thinking of the TransferUtility (which is the SDK's replacement for Transfer Manager, and what Amplify is currently using behind the scenes for many of its use cases)?